components/Modal/index.js

```diff
@@ -10,11 +10,12 @@ const styleModalBox = {
 	top: "50%",
 	left: "50%",
 	transform: "translate(-50%, -50%)",
-	width: "50%",
+	width: "80%",
```

Comment:
I cannot comment on line 8: should it be `styledModalBox` and a `d` is missing there?